Sources/Clappr_iOS/Classes/Plugin/Playback/AVFoundationPlayback.swift

```diff
@@ -574,6 +574,10 @@ extension AVFoundationPlayback {
         return position
     }
 
+    open var currentDate: Date? {
```

Comment:
Hey guys,

If we need this information available for other `Plugins` then it should be added in the [Playback.swift](https://github.com/clappr/clappr-ios/blob/dev/Sources/Clappr_iOS/Classes/Base/Playback.swift) class.

The access for any information should never cast the Playback to a specific implementation like:
❎ Indicating that we are using not following our architecture:
```
let playback = self.currentPlayback as? AvFoundationPlayback
let date = playback?.currentDate
```

Reply:
Done!